src/Titles.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class Titles extends Component {
+    render() {
+        const titles = ["Make", "Model", "Model year", "Color", "Dealer e-mail"];
+        const tdStyle = {padding: "0 20px", backgroundColor: "#7f7f7f", fontWeight: "bold"}
+        return(
+               titles.map(title => <td style={tdStyle}> {title} </td>)
```

Comment:
- Each child in an array or iterator should have a unique "key" prop. See https://fb.me/react-warning-keys for more information. There is a good value to be used here as the `key`.
- Spaces around `{title}` are redundant.